jekyll/_cci2/workflows.md

```diff
@@ -90,6 +91,36 @@ workflows:
 ```
 
 The dependencies are defined by setting the `requires:` key as shown. The `deploy:` job will not run until the `build` and `test1` and `test2` jobs complete successfully. A job must wait until all upstream jobs in the dependency graph have run. So, the `deploy` job waits for the `test2` job, the `test2` job waits for the `test1` job and the `test1` job waits for the `build` job.
+
+## Holding a Workflow for a Manual Approval
+
+Workflows may be configured to wait for manual approval of a job before continuing by using the `type: approval` key. The `type: approval` key is a special job type that is **only** added under in your `workflow` key. This enables you to configure a job with `type:approval` in the workflow before a set of parallel jobs that must all wait for manual approval. Jobs run in the order defined until the workflow processes a job with the `type: approval` key followed by a job on which it depends as in the following `config.yml` example:
```

Comment:
> Workflows may be configured to wait for manual approval of a job before continuing by using the `type: approval` key.

This is quite passive. How about:

> You can add a `type: approval` job to your workflow to make other jobs wait for manual approval before continuing.

---

> This enables you to configure a job with `type: approval` in the workflow before a set of parallel jobs that must all wait for manual approval.

This isn't really right: this says that the configuration enables you to add configuration. I'm not sure a sentence is actually necessary here, though; I'd just cut it. The example below is going to get the point across best.

---

> Jobs run in the order defined until the workflow processes a job with the `type: approval` key followed by a job on which it depends as in the following `config.yml` example:

This is actually incorrect: jobs don't run in the order defined at all, they run in dependency order. How about:

> Any job which depends on an Approval job will wait to run until that Approval job has been manually approved, as in the following `config.yml` example: